metrics-operator/controllers/common/providers/dynatrace/dynatrace.go

```diff
@@ -43,7 +43,13 @@ type DynatraceData struct {
 func (d *KeptnDynatraceProvider) EvaluateQuery(ctx context.Context, metric metricsapi.KeptnMetric, provider metricsapi.KeptnMetricsProvider) (string, []byte, error) {
 	baseURL := d.normalizeAPIURL(provider.Spec.TargetServer)
 	query := url.QueryEscape(metric.Spec.Query)
-	qURL := baseURL + "v2/metrics/query?metricSelector=" + query
+	interval := metric.Spec.Range.Interval
```

Comment:
please check if `metric.Spec.Range` is not `nil` first, otherwise this will lead to nil pointer accesses when there is no interval set in a `KeptnMetric`.

Reply:
Thank you, fixed it.